Kitodo/src/main/webapp/WEB-INF/templates/includes/processEdit/details.xhtml

```diff
@@ -46,6 +47,12 @@
                     <p:ajax event="change" oncomplete="toggleSave()"/>
                 </p:selectOneMenu>
             </div>
+
+            <div jsf:rendered="#{SecurityAccessController.hasAuthorityToAssignOcrdWorkflow()}">
```

Comment:
I didn't know JSF attributes can be used in standard html tags. Very interesting! 👍